### ISSUE TITLE
Replace database timeout error with relogin

### ISSUE THREAD

**Issue by nickbe** [opensupports/opensupports#420]
If the sesssion has a timeout we currently see something like this:
"An error occurred while trying to retrieve tickets"
But this is actually a timeout. So I suggest the user is simply re-routed to his login where he's asked to re-login because of the timout.

**Comment by peerres** [comment]
I agree, I've even been studying the code in order to manually change this redirection, because the user often does not understand that the session has expired and reports a bug in the system.
